modules/menus/dashboard/profile/index.ts

```diff
@@ -24,7 +25,14 @@ const Profile = () => {
                     Widget.Icon({
                         hpack: "center",
                         class_name: "profile-picture",
-                        icon: image.bind("value"),
+                        icon: image.bind("value").as(i => {
+                            try{
+                                GdkPixbuf.Pixbuf.new_from_file(i)
+                                return i
+                            } catch {
+                                return "avatar-default";
```

Comment:
```suggestion
                                return "avatar-default-symbolic";
```
Lets use symbolic icons as they can be styled and colored.